MvvmCross/Platforms/Wpf/Core/MvxWpfSetup.cs

```diff
@@ -97,10 +91,14 @@ protected override IMvxNameMapping CreateViewToViewModelNaming()
             return new MvxPostfixAwareViewToViewModelNameMapping("View", "Control");
         }
 
+        protected override void InitializeFirstChance()
+        {
```

Comment:
See my other comment - we should call the base method ie base.InitializeFirstChance()

Reply:
@nickrandolph my mistake - this was an oversight when I was removing some test code I must have removed the base calls as well - its fixed now.